app/admin/cycles.rb

```diff
@@ -0,0 +1,17 @@
+ActiveAdmin.register Cycle do
+  permit_params :curriculum,
+                :name,
+                :order_number,
+                :learning_goals_description,
+                :challenge_description,
+                :boilerplate_url
```

Comment:
Mataria el `boilerplate_url`

Reply:
I think that to remove it is more expensive than just leaving it there. Since if we are removing it because it's not used we should find and remove it everywhere if we want to be consistent